components/layout_2020/flow/mod.rs

```diff
@@ -1632,23 +1630,23 @@ impl PlacementState {
                     self.current_margin
                         .adjoin_assign(&fragment_block_margins.end);
                 } else {
-                    self.current_block_direction_position +=
-                        self.current_margin.solve() + fragment_block_size.into();
+                    self.current_block_direction_position += fragment_block_size.into();
```

Comment:
You've modified the code here. You are no longer adding `self.current_margin.solve()`.